validator/lib/validateCmpt.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+var Cesium = require('cesium');
+var validateB3dm = require('../lib/validateB3dm');
+var validateI3dm = require('../lib/validateI3dm');
+var validatePnts = require('../lib/validatePnts');
+
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = validateCmpt;
+
+/**
+ * Checks if provided buffer has valid cmpt tile content
+ *
+ * @param {Buffer} content - A buffer containing the contents of a cmpt tile.
+ * @returns {Object} An object with two parameters - (1) a boolean for whether the tile is a valid cmpt tile
+ *                                                   (2) a message to indicate which tile field is invalid, if any
+ */
+function validateCmpt(content) {
+    if (!defined(content)) {
+        throw new DeveloperError('cmpt content must be defined');
+    }
+
+    if (!Buffer.isBuffer(content)) {
+        throw new DeveloperError('content must be of type buffer');
+    }
+
+    if (content.length < 16) {
```

Comment:
Save `16` off as a variable called `headerByteLength`. This will make it easier to follow.